Automattic/jetpack
# AI Inline Extensions: Support custom behavior

Related with #37590 

## Proposed changes:

* Support a custom behavior passing through a function.
* Export actions through context if isn't a children block.
* Mark `list-item` as `childrenBlock`.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






* Go to some block class handler (ex. paragraph).
* Add a method to handle the behavior (ex. `handleBehavior`).
* Expose it on `constructor` after `super` call: `this.behavior = this.handleBehavior`.
* You should be able to `log` the parameters (with `context`).
* If you do the same on `list-item` you need to receive the parent context (`list`).
